packages/vscode-extension/package.json

```diff
@@ -186,6 +186,26 @@
                 "type": "string",
                 "description": "Location of Metro config relative to the workspace. This is used for using custom configs for e.g. development."
               },
+              "buildScript": {
+                "type": "object",
+                "description": "Script used to build app or fetch app from known location. Executed as part of building process. Receives 'ios' or 'android' as first argument and should return a path to built app.",
```

Comment:
Expand on this. The script will not "return path", it prints it to stdout. Add note that this is "advanced" options for setups where non-standard build instructions are used